lib/lang/greek/greek-language-dataset.js

```diff
@@ -436,10 +510,17 @@ export default class GreekLanguageDataset extends LanguageDataset {
     return this.pronounGroupingLemmas.has(grammarClass) ? this.pronounGroupingLemmas.get(grammarClass) : []
   }
 
+  getNumeralGroupingLemmas () {
+    return this.numeralGroupingLemmas
+  }
+
   static getObligatoryMatchList (inflection) {
     if (inflection.hasFeatureValue(Feature.types.part, Constants.POFS_PRONOUN)) {
       // If it is a pronoun, it must match a grammatical class
       return [Feature.types.grmClass]
+    } else if (inflection.hasFeatureValue(Feature.types.part, Constants.POFS_NUMERAL)) {
+      // If it is a pronoun, it must match a grammatical class
```

Comment:
Sorry, here is a comment I inserted by mistake via copy and paste, can you please remove it?